### ISSUE TITLE
Listing TDs gives 405 ERROR - version 10

### ISSUE THREAD

**Issue by karaleina** [thingweb/thingweb-directory#39]
I created Postman client based on Swagger documentation (api.json file) and tried to list available things (expecting empty list) and I got 405 ERROR. 

 (I set up version 10 according to this guideline https://github.com/thingweb/thingweb-directory/issues/33)

**Comment by 6d77** [comment]
Indeed this seems not to be implemented and one of the things, where the Swagger file is definitely not in line with the implementation. A pure GET on `/td` is not supported. 

To list all things, you can use the web interface at `/` or the query at `td-lookup/sem`, which however not only returns the names of the things, but all thing descriptions.

**Comment by karaleina** [comment]
Indeed, I see that using web interface it works, but unfortunately, when I make GET on 'http://localhost:8080/td-lookup/sem', as you suggested, I get 406 ERROR. 

Is there any example of how to make query at 'td-lookup/sem' (e.g. via curl or Postman)?

**Comment by karaleina** [comment]
Thank you, it is possible to query td-lookup/sem instead.  Referring to Swagger documentation, I needed to change header _Accept_ to _application/td+json_.
